src/main/python/hydra/lib/runtestbase.py

```diff
@@ -6,17 +6,28 @@
 import time
 import sys
 from hydra.lib import appserver, mmapi, util
+from ConfigParser import ConfigParser
 
 l = util.createlogger('runTestBase', logging.INFO)
+from marathon.models import MarathonApp, MarathonConstraint
 # l.setLevel(logging.DEBUG)
 
 
 class RunTestBase(object):
-    def __init__(self, test_name, config):
+    def __init__(self, test_name, config = None, config_filename = None,
```

Comment:
Not sure if I understand, Are you trying to suggest adding arguments that don't have default. 
In general it will solve the problem of modifying this particular line but those arguments still need to be handled. This way there is a lint time check that flags the use of arguments that are not supported.